### PR TITLE
create new company admin migration script

### PR DESCRIPTION
**Pull request by kotky**
Created new company admin script for adding the new company admin group on mckinsey academy and adding that group to all organizations

### CODE REVIEW

File: lms/djangoapps/api_manager/management/commands/add_new_permissions.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import json
+from django.core.exceptions import ObjectDoesNotExist
+
+from api_manager.models import GroupRelationship, GroupProfile
+from django.contrib.auth.models import User, Group
+from organizations.models import Organization, OrganizationGroupUser
+
+class Command(BaseCommand):
+    help = """Repair users that are assistants and observers on every course
+example:
```
Comment (vkaracic) [inline lms/djangoapps/api_manager/management/commands/add_new_permissions.py]: Indent properly.

File: lms/djangoapps/api_manager/management/commands/add_new_permissions.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+"""
+
+    def handle(self, *args, **options):
+        
+        group_type = "permission"
+        original_group_name = "mcka_role_company_admin"
+        
+        try:
+            mcka_company_admin = Group.objects.get(name__icontains=original_group_name)
+            print "Found company admin group ("+original_group_name+"), id: " + str(mcka_company_admin.id)
```
Comment (vkaracic) [inline lms/djangoapps/api_manager/management/commands/add_new_permissions.py]: Use ``logging`` and either ``%s`` or ``format`` for formatting strings.

File: lms/djangoapps/api_manager/management/commands/add_new_permissions.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        
+        except ObjectDoesNotExist:
```
Comment (vkaracic) [inline lms/djangoapps/api_manager/management/commands/add_new_permissions.py]: ``Group.DoesNotExist``.

File: lms/djangoapps/api_manager/management/commands/add_new_permissions.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            print "Creating new company admin group (mcka_role_company_admin)..."
+            mcka_company_admin = Group.objects.create(name=str(uuid.uuid4()))
+            mcka_company_admin.name = '{:04d}: {}'.format(mcka_company_admin.id, original_group_name)
```
Comment (vkaracic) [inline lms/djangoapps/api_manager/management/commands/add_new_permissions.py]: Use ``mcka_company_admin.name`` if you're using the name generated in the line above.

**Comment by ziafazal** [comment]
@kotky  over all looks good we need to take care of these 3 things.

1. We have moved api_manager app into a separate [repo](https://github.com/edx-solutions/api-integration)  so this PR needs to be moved there.
2. We need to write a reverse command for operations done in this command
3. We should make sure Circle CI buid is passing

**Comment by kotky** [comment]
@ziafazal I have moved the PR on new repo, I think that it is fine, I have only changed imports name because they are new one.

**Pull request closed without merging**